### PR TITLE
KEYCLOAK-8836 Add test to check product name on welcome page

### PR DESCRIPTION
**Pull request by Aboullos**
Add test to check the presence of the correct product name on the welcome page

### CODE REVIEW

**Review by ssilvert — COMMENTED** [review]
LGTM.  If @vmuzikar agrees then we can merge.

**Review by stianst — CHANGES_REQUESTED** [review]

File: testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/welcomepage/WelcomePageTest.java [hunk 1]
```diff
@@ -161,4 +161,14 @@ public void test_5_AccessCreatedAdminAccount() throws Exception {
                 driver.getPageSource().contains("Invalid username or password."));
     }
 
+    @Test
+    public void test_6_CheckProductNameOnWelcomePage() {
+        welcomePage.navigateTo();
+
+        String actualMessage = welcomePage.getWelcomeMessage();
+        String expectedMessage = suiteContext.getAuthServerInfo().isEAP() ? "Red Hat Single Sign-On" : "Keycloak";
```
[the remark this answers is not in the document]
Comment (stianst) [reply testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/welcomepage/WelcomePageTest.java]: Do we have a way to get profile information on the test side?

[the remark this answers is not in the document]
Comment (vmuzikar) [reply testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/welcomepage/WelcomePageTest.java]: @stianst I don't know of a better approach (and I use it myself like that too :D).

[the remark this answers is not in the document]
Comment (stianst) [reply testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/welcomepage/WelcomePageTest.java]: Oki dokey

**Review by stianst — APPROVED** [review]
LGTM

**Review by vmuzikar — APPROVED** [review]
LGTM, tests passed with both KC and RH-SSO.
Thanks @Aboullos!

**Pull request merged**